src/main/java/core/basesyntax/strategy/Action.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface Action {
+    void action(FruitTransaction fruitTransaction);
```

Comment:
```suggestion
    void processOperation(FruitTransaction fruitTransaction);
```
use verbs for method names